staging/src/k8s.io/kube-aggregator/pkg/apiserver/metrics.go

```diff
@@ -46,7 +50,36 @@ var x509InsecureSHA1Counter = metrics.NewCounter(
 	},
 )
 
+var (
+	extensionApiserverRequestCounter = metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Name:           "extension_apiserver_requests_total",
```

Comment:
Could you extract `extension_apiserver` from the name and put it in the `Subsytem` field? Similar to what is done in https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go#L63. This makes sure that withing this file all the metrics will share the same prefix.